python/pyspark/mllib/util.py

```diff
@@ -169,6 +169,22 @@ def loadLabeledPoints(sc, path, minPartitions=None):
         minPartitions = minPartitions or min(sc.defaultParallelism, 2)
         return callMLlibFunc("loadLabeledPoints", sc, path, minPartitions)
 
+    @staticmethod
+    def appendBias(data):
+        """
+        Returns a new vector with `1.0` (bias) appended to the input vector.
```

Comment:
As long as we're at it, could you please make this more specific: "appended to the end of the input vector"
And could you please add it to the Scala version doc too?